robottelo/constants.py

```diff
@@ -1224,3 +1117,414 @@
     'Installable',
     '7/27/16'
 ]
+
+# To create custom role for bugzilla's
+# Append bugzilla id to dict, also see utils.get_perms_by_bz()
+PERMISSIONS_WITH_BZ = {
+    None: [
+        {'name': 'access_dashboard', 'bz': [1306359]},
+        {'name': 'access_settings', 'bz': [1306359]},
+        {'name': 'app_root', 'bz': []},
```

Comment:
implementing https://github.com/SatelliteQE/robottelo/pull/4215/files#r97305883 would make 'bz' key with empty list unnecessary (dict.get('key') would return None if the 'key' does not exist so you wouldn't get an exception)